imap_processing/tests/test_decom.py

```diff
@@ -0,0 +1,61 @@
+"""Tests aspects of the decommutation process that is generic to all instruments"""
+
+import glob
+
+import pytest
+
+from imap_processing import imap_module_directory
+
+xtce_documents = glob.glob(f"{imap_module_directory}/*/packet_definitions/*.xml")
+xtce_documents = [
+    item for item in xtce_documents if "idex_packet_definition.xml" not in item
+]  # Temporary until IDEX packet definition is fixed
+
+
+@pytest.mark.parametrize("xtce_document", xtce_documents)
+def test_ccsds_header(xtce_document):
+    """Test if the XTCE document contains the proper CCSDS header information"""
+
+    print(f"\nTesting {xtce_document}")
+
+    # Define what is expected in the XTCE document
+    header_keys = [
+        "VERSION",
+        "TYPE",
+        "SEC_HDR_FLG",
+        "PKT_APID",
+        "SEG_FLGS",
```

Comment:
```suggestion
        "SEQ_FLGS",
```
Do we have a typo in all of these?

Reply:
It turns out we do have a bunch of packet definition files that have this type, as @GFMoraga pointed out to me. He is going to fix the files in a separate PR.